ddr/tools/blob_reader/blob_reader.cpp

```diff
@@ -299,7 +299,12 @@ main(int argc, char *argv[])
 				/* The format of the printed list is:
 				 * #: <offset in string data> [<string length>] <string data>
 				 */
-				omrfile_printf(OMRPORT_TTY_OUT, "%5u: %8zx [%zu] %.*s\n", stringNum, (uintptr_t)(currentString - stringDataStart), blobString->length, blobString->length, blobString->data);
+				omrfile_printf(OMRPORT_TTY_OUT, "%5u: %8zx [%u] %.*s\n",
+						stringNum,
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line

The parameter list should start on a new line.
```suggestion
				omrfile_printf(
						OMRPORT_TTY_OUT, "%5u: %8zx [%u] %.*s\n",
						stringNum,
```

Reply:
This is consistent with all the other uses of `omrfile_printf()`, or are you asking me to change all of them?

Reply:
I will accept as-is since the coding standard allows using the old format: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#old-code.